<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PisImportViewClass</class>
 <widget class="QWidget" name="PisImportViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PisImportView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Внимание! 
Тази услуга предоставя възможност за сваляне на амбулаторните листи и пациентите, които сте подавали с месечните отчети в ПИС. При сваляне на листи, които са отчетени и в НЗИС, информацията в тях ще бъде непълна! Не правете корекции върху тези листи, тъй като това ще доведе до запис на грешни или дублирани данни в пациентското досие в НЗИС!

- Оралният статус се импортира по номенклатурата на НЗОК
- Не се импортират алергии, настоящи и минали заболявания
- Не се импортират издадени направления
- Не се импортира настоящият адрес на пациента (ЕКАТТЕ)
- Не се импортират чужденици с идентификатор тип 4 (граждани на ЕС)
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Импортирай амбулаторни листи от ПИС</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
